packages/widgets/src/lib/components/InferenceWidget/shared/WidgetHeader/WidgetHeader.svelte

```diff
@@ -66,15 +66,23 @@
 </div>
 <div class="mb-0.5 flex w-full max-w-full flex-wrap items-center justify-between text-sm text-gray-500">
 	{#if pipeline && task}
-		<a
-			class={TASKS_DATA[task] ? "hover:underline" : undefined}
-			href={TASKS_DATA[task] ? `/tasks/${task}` : undefined}
-			target="_blank"
-			title={TASKS_DATA[task] ? `Learn more about ${task}` : undefined}
-		>
-			<PipelineTag classNames="mr-2 mb-1.5" {pipeline} />
-		</a>
+		<div class="flex gap-4 items-center mb-1.5">
+			<a
+				class={TASKS_DATA[task] ? "hover:underline" : undefined}
+				href={TASKS_DATA[task] ? `/tasks/${task}` : undefined}
+				target="_blank"
+				title={TASKS_DATA[task] ? `Learn more about ${task}` : undefined}
+			>
+				<PipelineTag {pipeline} />
+			</a>
+			{#if $tgiSupportedModels?.has(model.id)}
+				<p class="text-xs text-gray-400">
+					Streaming with <a href="https://huggingface.co/docs/text-generation-inference" class="underline">TGI</a>
```

Comment:
```suggestion
					Streaming with <a href="https://huggingface.co/docs/text-generation-inference" class="hover:underline">TGI</a>
```

made it to `hover:underline`

Reply:
cc @gary149